RockWeb/Blocks/Finance/TransactionEntryV2.ascx

```diff
@@ -53,40 +48,53 @@
                         <Rock:CampusAccountAmountPicker ID="caapPromptForAccountAmounts" runat="server" />
 
                         <asp:Panel ID="pnlScheduledTransaction" runat="server">
+                            <div class="form-group">
+                                <Rock:RockDropDownList ID="ddlFrequency" runat="server" CssClass="input-lg" AutoPostBack="true" OnSelectedIndexChanged="ddlFrequency_SelectedIndexChanged" />
+                            </div>
+
+                            <div class="form-horizontal">
+                                <asp:Panel ID="pnlSavedAccounts" runat="server" class="form-group" Visible="false">
+                                    <label class="control-label col-xs-5"  style="text-align:left;" for="<%= ddlPersonSavedAccount.ClientID %>">Giving Method</label>
+                                    <div class="control-wrapper col-xs-7">
+                                        <Rock:RockDropDownList ID="ddlPersonSavedAccount" runat="server" CssClass="pull-right" />
+                                    </div>
+                                </asp:Panel>
 
-                            <Rock:RockDropDownList ID="ddlFrequency" runat="server" FormGroupCssClass=" margin-t-md" AutoPostBack="true" OnSelectedIndexChanged="ddlFrequency_SelectedIndexChanged" />
+                                <div class="form-group">
+                                    <label class="control-label col-xs-5" style="text-align:left;" for="<%= dtpStartDate.ClientID %>"><asp:Literal ID="lStartDateLabel" runat="server" /></label>
+                                    <div class="control-wrapper col-xs-7">
+                                        <Rock:DatePicker ID="dtpStartDate" runat="server" AllowPastDateSelection="false" CssClass="pull-right" />
```

Comment:
@richarddubay this is where that `pull-right` is that probably shouldn't be there - unless there's a reason for it to be right-aligned. Just fwiw.

Reply:
👍